frontend/src/scenes/insights/ActionFilter/ActionFilter.js

```diff
@@ -66,7 +66,7 @@ export function ActionFilter({
     return (
         <div>
             {localFilters ? (
-                sortable ? (
+                sortable !== undefined ? (
```

Comment:
This will always evaluate to true for both true/false. Is that intentional?

Reply:
Hm I guess the issue is we were passing `sortable` without any params and expecting it to result in true. What's the best way of doing this?

Reply:
What's the bug here? Passing sortable without params [should default to true](https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true) so the check should work. I also double checked manually in QA and the sortable component appears as expected when there are multiple entities for funnels

Reply:
That's super weird, this didn't work for me locally but now does. I'll close this